Javascript/reset.js

```diff
@@ -1,625 +1,650 @@
+    }
+    document.getElementById("resetcurrency2").textContent = "-" + format(player.acceleratorBoostCost)
+    document.getElementById("resetobtainium").src = ""
+    document.getElementById("resetobtainium2").textContent = ""
+    document.getElementById("resetinfo").textContent = "Reset Coin Producers/Upgrades, Crystals and Diamonds in order to increase the power of your Accelerators. Required: " + format(player.prestigePoints) + "/" + format(player.acceleratorBoostCost) + " Diamonds."
+    if (player.prestigePoints.greaterThanOrEqualTo(player.acceleratorBoostCost)) {
+      document.getElementById("resetinfo").style.color = "limegreen"
+    } else {
+      document.getElementById("resetinfo").style.color = "crimson"
+    }
+  }
+  if (i == 6) {
+    var s = player.currentChallengeRein
+    let goal = "transcendShards";
+    let goaldesc = " Mythos Shards"
+    if (player.currentChallengeRein == "nine" || player.currentChallengeRein == "ten") {
+      goal = "coins";
+      goaldesc = " Coins"
+    }
+    color = 'red'
+    document.getElementById("resetobtainium").src = ""
+    document.getElementById("resetcurrency1").src = ""
+    document.getElementById("resetcurrency2").textContent = ""
+    document.getElementById("resetobtainium2").textContent = ""
+    if (player.currentChallengeRein !== "") {
+      if (reincarnationChallengeBeaten()) {
+        document.getElementById("resetinfo").style.color = "limegreen"
+      } else {
+        document.getElementById("resetinfo").style.color = "crimson"
+      }
+      document.getElementById("resetinfo").textContent = "Are you done or tired of being in your challenge? Click to leave challenge " + s + ". Progress: " + format(player[goal]) + "/" + format(getCurrentReincarnationChallengeGoal()) + goaldesc + ". TIME SPENT: " + format(player.reincarnationcounter) + " Seconds."
+    }
+    if (player.currentChallengeRein == "") {
+      document.getElementById("resetinfo").textContent = "You're not in a reincarnation challenge right now. Why would you need to leave it?"
+      document.getElementById("resetinfo").style.color = "crimson"
+    }
+  }
+  if (i == 7) {
+    document.getElementById("resetobtainium").src = ""
+    document.getElementById("resetcurrency1").src = ""
+    document.getElementById("resetcurrency2").textContent = ""
+    document.getElementById("resetobtainium2").textContent = ""
+    document.getElementById("resetinfo").style.color = "gold"
+    document.getElementById("resetinfo").textContent = "You've reached the end of v1.010. This will be the [ASCEND] reset tier!"
+
+  }
+
+
+  document.getElementById("resetofferings2").textContent = "+" + format(q)
 
 }
 
 
 
 function updateAutoReset(i) {
-    if (i == 1) {
-        var t = document.getElementById("prestigeamount").value
-         if (t >= 0){player.prestigeamount = t;}
-         else {player.prestigeamount = 0;}
+  if (i == 1) {
+    var t = document.getElementById("prestigeamount").value

```

Comment:
vars can be changed to either lets or consts. In this case, they would be consts.

Reply:
i do not know the reasoning on why we do not use vars, we just do not.

Reply:
Done